puppet/modules/profiles/manifests/repo/rpm.pp

```diff
@@ -0,0 +1,20 @@
+# @summary A profile for the rpm repo machines
+#
+# @param https
+#   Whether to enable HTTPS. This is typically wanted but can only be enabled
+#   in a 2 pass setup. First Apache needs to run for Letsencrypt to function.
+#   Then Letsencrypt can be enabled. Also useful to turn off in test setups.
+class profiles::repo::rpm (
+  Boolean $https = true,
+  String[1] $stable = '3.11',
```

Comment:
Now that we're duplicating this, I think it makes sense to define the actual value in Hiera to avoid them going out of sync. I think it's also nicer to update Hiera files than class defaults (since it's really data we're modifying).

Reply:
Same discussion: https://github.com/theforeman/foreman-infra/pull/2108#discussion_r1734618389

Reply:
https://github.com/theforeman/foreman-infra/pull/2119 was implented, so you can use that now.